### ISSUE TITLE
Wither Skeletons do not spawn in "Hell" biomes/Nether Islands.

### ISSUE THREAD

**Issue by KaloHG** [CivClassic/SimpleAdminHacks#33]
Major issue that prevents new players from going through the tech tree, notably the Advanced EXP Cauldron and Beacon recipes require wither skeleton heads.

Probably just got left behind on accident with humbug.

**Comment by Maxopoly** [comment]
Should they do that? I was under the impression they only spawn from portals

**Comment by wingzero54** [comment]
Yes they were able to be farmed in nether islands. I don't recall ever seeing a wither skeleton come out of a portal.

**Comment by KaloHG** [comment]
Pull request #34 restores original functionality
